src/python-common/ceph/deployment/service_spec.py

```diff
@@ -706,6 +711,7 @@ def __init__(self,
                  rgw_frontend_type: Optional[str] = None,
                  unmanaged: bool = False,
                  ssl: bool = False,
+                 networks: Optional[List[str]] = None,
```

Comment:
I think we have to make this a real property of the base class `ServiceSpec`. We need to be able to make the daemons bind to specific IPs not just for RGWs, but for all other services as well.

I think we can start implementing this for RGWs only, but I think we should attach `netowrks` to the base class already.

Reply:
Hmm.  Thinking about this a bit more, there is an argument to be made that this belongs in PlacementSpec.

- It's closer to the HostPlacementSpec, which also has a networks field.  The handling for the two could be converged...
- It consolidates the input for the scheduling in PlacementSpec, making HostAssignment a bit more modular.
- We have to filter the candidate hosts by this when scheduling

The same would apply to port, though.  Which makes it weirder to have default ports associated with service types.